x-pack/plugins/rac/scripts/generate_mapping/mappings.js

```diff
@@ -0,0 +1,440 @@
+            output_index: {
+              type: 'keyword',
+            },
+            description: {
+              type: 'keyword',
+            },
+            from: {
+              type: 'keyword',
+            },
+            immutable: {
+              type: 'keyword',
+            },
+            index: {
+              type: 'keyword',
+            },
+            interval: {
+              type: 'keyword',
+            },
+            language: {
+              type: 'keyword',
+            },
+            license: {
+              type: 'keyword',
+            },
+            name: {
+              type: 'keyword',
+            },
+            rule_name_override: {
+              type: 'keyword',
+            },
+            query: {
+              type: 'keyword',
+            },
+            references: {
+              type: 'keyword',
+            },
+            severity: {
+              type: 'keyword',
+            },
+            severity_mapping: {
+              properties: {
+                field: {
+                  type: 'keyword',
+                },
+                operator: {
+                  type: 'keyword',
+                },
+                value: {
+                  type: 'keyword',
+                },
+                severity: {
+                  type: 'keyword',
+                },
+              },
+            },
+            tags: {
+              type: 'keyword',
+            },
+            threat: {
+              properties: {
+                framework: {
+                  type: 'keyword',
+                },
+                tactic: {
+                  properties: {
+                    id: {
+                      type: 'keyword',
+                    },
+                    name: {
+                      type: 'keyword',
+                    },
+                    reference: {
+                      type: 'keyword',
+                    },
+                  },
+                },
+                technique: {
+                  properties: {
+                    id: {
+                      type: 'keyword',
+                    },
+                    name: {
+                      type: 'keyword',
+                    },
+                    reference: {
+                      type: 'keyword',
+                    },
+                    subtechnique: {
+                      properties: {
+                        id: {
+                          type: 'keyword',
+                        },
+                        name: {
+                          type: 'keyword',
+                        },
+                        reference: {
+                          type: 'keyword',
+                        },
+                      },
+                    },
+                  },
+                },
+              },
+            },
+            threshold: {
+              properties: {
+                field: {
+                  type: 'keyword',
+                },
+                value: {
```

Comment:
we've not added a notion of a fixed type for thresholds, because "IT depends".  The index threshold has comparators "between" and "not between" which take two threshold values.  The geo containment alert threshold is some geo-related value - I believe a shape, but perhaps it's a set of points, or ??? - it's unlikely to be captured as a `float` though.